packages/expo-location/src/Location.ts

```diff
@@ -159,8 +171,39 @@ export async function enableNetworkProviderAsync(): Promise<void> {
 }
 
 export async function getCurrentPositionAsync(
-  options: LocationOptions = {}
+  options: LocationOptions = {},
+  methodOptions: LocationMethod = { type: LocationMethodType.Fast }
 ): Promise<LocationData> {
+  console.log(methodOptions);
+  if (methodOptions.type === LocationMethodType.Fast) {
+    console.log("fast");
+    return new Promise<LocationData>(async (resolve, reject) => {
+      try {
+        let bestPosition: LocationData | null = null;
+        let current = 0;
+        let { tries, minAccuracyRadius } = {
```

Comment:
In general follow these guidelines:
1. use `const` as default value binding
2. use `let` if binding should change (or primitive value should and would be changed)
3. do not use `var` in ES5 and above 😄 

Summing up -> change all `let`s to `const`s